tests/post_training/pipelines/base.py

```diff
@@ -131,6 +131,8 @@ class RunInfo:
     metric_value: Optional[float] = None
     metric_diff: Optional[float] = None
     num_fq_nodes: Optional[float] = None
+    num_int8: Optional[float] = None
+    num_int4: Optional[float] = None
```

Comment:
`RunInfo` is becoming too big and contains fields that are needed for a specific pipeline only. 
It starts violating [the ISP principle of SOLID](https://en.wikipedia.org/wiki/Interface_segregation_principle)
Please add `num_compress_nodes` field that stores a class-container for pipeline-specific numbers. For `PTQTestPipeline` 
Instead of 3 integer fields, add class hierarchy similar to `StatsFromOutput`, `PTQTimeStats` and `WCTimeStats`.